afs-spring-server/src/main/java/com/powsybl/afs/server/StorageServer.java

```diff
@@ -0,0 +1,605 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.server;
+
+import com.powsybl.afs.*;
+import com.powsybl.afs.storage.*;
+import com.powsybl.afs.storage.buffer.*;
+import com.powsybl.afs.ws.server.utils.JwtTokenNeeded;
+import com.powsybl.timeseries.DoubleDataChunk;
+import com.powsybl.timeseries.StringDataChunk;
+import com.powsybl.timeseries.TimeSeriesMetadata;
+import io.swagger.annotations.*;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.*;
+
+@JwtTokenNeeded
```

Comment:
I would prefer not to use it here : I think it's a security choice which is the responsibility of the application, not of the controller itself : today in our project we don't use JWT authentication, for example.

Actually I am not sure it does something ? (does spring support this `@Provider` mechanism ?)

Reply:
indeed, with spring the authentication filter defined in the application. so here it is to be removed

Reply:
However, we should create another annotation name which isn't used by the JEE JwtTokenNeededFilter

Reply:
Not sure if I understand everything correctly, but you are saying this spring controller is used by another application? Maybe you can create a class an extend this spring controller and add the annotation there and remove it here ?

Reply:
I would vote up for that solution, if that's fin with you @pl-buiquang . Athough the annotation is harmless, it is also misleading (and I guess not usable without a specific filter which is in your project ?).

Reply:
ok for me